### ISSUE TITLE
Launcher can not load mod versions, changelog is loaded correctly but post request to GDLauncher api fails

### ISSUE THREAD

**Issue by pixartist** [killpowa/Twickt-Launcher#1251]
**Describe the bug**
Launcher fails to load version data for All the Mods 7

**To Reproduce**
1. Install ATM7
2. Go to manage -> modpack

**Expected behavior**
The dropdown lists available versions

**Possible solutions**
-

**Screenshots**
![image](https://user-images.githubusercontent.com/628738/149087386-b11a7e79-e684-4def-b347-cc55ab1063d2.png)
![image](https://user-images.githubusercontent.com/628738/149087414-9f1cf3f4-88d6-4f37-9959-9ad4b6808fbc.png)

**Operating System:**
- OS: Ubuntu
- Version: 21.10
- Java Version: 17.0.1 (Temurin)
- Minecraft Version: 1.18.1
- Mod Loader: Forge

**Additional context**
This also occurs on windows, I assume an API problem

**Comment by Eskaan** [comment]
Yes it is. It may be the same issue than #1154, but you can leave it opened as its another issue caused by the API.

**Comment by AshtakaOOf** [comment]
So hey @pixartist could you try the latest beta ? 
https://github.com/gorilla-devs/GDLauncher/releases/tag/v1.1.22-beta.1
